src/topp/MzTabExporter.cpp

```diff
@@ -712,7 +662,62 @@ namespace OpenMS
 
       return mztab;
     }
- 
+
+    // generate list of modifications from modified AASequence object. All passed fixed modifications are not reported (PRT and PEP section).
```

Comment:
"modified AASequence object"? Modified how? Or do you mean potentially storing a sequence with modifications?